src/Ledger/Gov.lagda

```diff
@@ -110,13 +111,13 @@ data _⊢_⇀⦇_,GOV'⦈_ where
       (Γ , k) ⊢ s ⇀⦇ sig ,GOV'⦈ addVote s aid voter v
 
   GOV-Propose : ∀ {x} → let
-      open GovEnv Γ; open PParams pparams hiding (a)
+      open GovEnv Γ
```

Comment:
Remove one instance of `open` for record `pparams` at the expense of having to write `pparams` a couple of times below.